### PR TITLE
Add missing `/`, fixes tests on sqlite

### PR DESCRIPTION
**Pull request by jadb**

### CODE REVIEW

**Comment by ceeram** [comment]
Makes the tests at least run for me, so good job on that one. Still getting another fatal though locally.
Here is the output of phpunit of both master and of this PR:

https://gist.github.com/ceeram/020b3c102f196c29790c

Haven't investigated this error further yet, `General error: 17 database schema has changed`

**Comment by ceeram** [comment]
the DROP TABLES in RequestTableTest seem to cause the schema error. If run all tests without RequestTableTest then all passes.

File: tests/bootstrap.php [hunk 1]
```diff
@@ -88,7 +88,7 @@
 
 // Ensure default test connection is defined
 if (!getenv('db_dsn')) {
-    putenv('db_dsn=sqlite://' . TMP . 'debug_kit_test.sqlite');
+    putenv('db_dsn=sqlite:///' . TMP . 'debug_kit_test.sqlite');
```
Comment (markstory) [inline tests/bootstrap.php]: TMP normally starts with `/`

Comment (ceeram) [inline tests/bootstrap.php]: according to php.net only a single / would do, but the gist shows it really needs the extra /

Comment (markstory) [inline tests/bootstrap.php]: Ok.

Comment (ceeram) [inline tests/bootstrap.php]: In fact, putenv('db_dsn=sqlite:/' . TMP . 'debug_kit_test.sqlite'); would have worked too.
So it works with 1 or 3 slashes, just not with 2

Comment (markstory) [inline tests/bootstrap.php]: Its like the holy hand grenade from monty python.

Comment (jadb) [inline tests/bootstrap.php]: I used 3 like in the tests: https://github.com/cakephp/cakephp/blob/master/tests/TestCase/Core/StaticConfigTraitTest.php#L222

**Pull request merged**